internal/scorecard/plugins/plugin_runner.go

```diff
@@ -133,6 +135,21 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 		if err = yaml.Unmarshal(yamlSpec, csv); err != nil {
 			return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("error getting ClusterServiceVersion: %v", err)
 		}
+
+		csvValidator := validation.ClusterServiceVersionValidator
+		results := csvValidator.Validate(csv)
+		for _, r := range results {
+			if len(r.Errors) > 0 {
+				var errorMsgs strings.Builder
+				for _, e := range r.Errors {
+					errorMsgs.WriteString(e.Error())
```

Comment:
If there are multiple errors, do we need a delimiter between each of the errors in the resulting string?

Is there a downside to logging the errors here and returning a simple error?
```go
for _, e := range r.Errors {
	log.Error(e.Error())
}
return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("error validating ClusterServiceVersion")
```

For follow-up, I'm thinking there's probably some redesign/refactoring to be done in this broader section of code that is making this more complex than is really necessary.

Reply:
I've added a delimiter for those log messages. I think this code goes away if we just require a bundle?  but I agree that scorecard logging versus what goes into error messages and the test results needs some refactoring in future PRs.